src/Runner.jl

```diff
@@ -331,6 +331,23 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         return "-mmacosx-version-min=$(macos_version(p))"
     end
 
+    function add_system_includedir(flags::Vector{String})
+        # GCC 4.8.5 for musl and clang on FreeBSD do not have `SYSROOT/usr/local` (which
+        # in our build environments is a symlink to $includedir = "$(prefix)/include")
+        # in the default list of include search directories. As a result, quite some
+        # builders that work fine everywhere else need to add something like
+        # `-I${includedir}` to their CPPFLAGS.
+        #
+        # To remove this annoyance, we simply add it here. We use `-isystem` instead
+        # of `-I` to mimic how this dir is treated on other operating systems. It
+        # is still not a perfect match, as the order of the default search directories
+        # differs, but this should not matter in practice.
+        #
+        # See also https://github.com/JuliaPackaging/Yggdrasil/issues/3949 and
+        # https://github.com/JuliaPackaging/Yggdrasil/pull/3969 for further details.
+        append!(flags, ["-isystem", raw"${includedir}"])
```

Comment:
I was looking at https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/9c15402d496a737a5d34b01409044a52c642f9be/src/BuildToolchains.jl#L114 and I realised this isn't right. We should use host_includedir if this is the host compiler (and I'm not sure we have an easy way to check that at the moment)